docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -425,8 +425,9 @@ Return low-level information on the container `id`
 			"Paused": false,
 			"Pid": 0,
 			"Restarting": false,
-			"Running": false,
-			"StartedAt": "2015-01-06T15:47:32.072697474Z"
+			"Running": true,
+			"StartedAt": "2015-01-06T15:47:32.072697474Z",
+			"Status": "running"
```

Comment:
Please note that these changes should go into `docker_remote_api_v1.21.md`, once https://github.com/docker/docker/pull/14981 is merged